using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -1068,6 +1153,16 @@ public interface AdobeCallback<T> {
 }
 ```
 
+**AdobeCallbackWithError**
+
+This class provides the interface to receive results or an error when the async APIs perform the requested action. When using this class, if the request cannot be completed after 500ms, the request is aborted and _fail_ is called.
```

Comment:
if the request cannot be completed within 500ms or if an unexpected error occurs, the request is aborted and the _fail_ method is called with the corresponding AdobeError.